docs/intro.rst

```diff
@@ -0,0 +1,111 @@
+   $ bin/zookeeper-server-start.sh config/zookeeper.properties
+   $ bin/kafka-server-start.sh config/server.properties
+   # Create testAvro topic in Kafka
+   $ bin/kafka-topics.sh --create --zookeeper localhost:2181 \
+         --topic testAvro --partitions 1 --replication-factor 1
+   $ cd ..
+
+   # Start Schema Registry. The default schema registry settings
+   # automatically work with the default settings of ZooKeeper and Kafka.
+   $ cd schema-registry
+   $ bin/schema-registry-start config/schema-registry.properties
+   $ cd ..
+
+   # Assuming that you have hadoop on your PATH, and make sure that
+   # schema.registry.url points to the correct address.
+   $ cd camus
+   $ hadoop jar confluent-camus-1.0-shaded.jar com.linkedin.camus.etl.kafka.CamusJob \
+     -D schema.registry.url=http://localhost:8081 -P etc/camus.properties
+
+Installation
+------------
+
+You can download Confluent Platform at .... And follow the instructions to install from
+rpm, deb or archives.
+
+Deployment
+----------
+Camus can be run from the command line. You will need to set some properties either by specifying a
+properties file on the classpath using ``-p`` (filename),or an external properties file using ``-P``
+(path to local file system, or to hdfs),
+or from the command line using ``-D property=value``.
+If the same property is set with multiple methods,
+the order of precedence is command-line properties, external properties file and
+classpath properties file. You can find a list of configurations in section Configuration
+options.
+
+
+.. sourcecode:: bash
+
+   $ hadoop jar confluent-camus-1.0-shaded.jar com.linkedin.camus.etl.kafka.CamusJob
+      -D <property=value>
+      -P <path to external properties file>
+      -p <path to properties file from classpath>
+
+Features
+--------
+
+Camus has the following features: 1. Automatic topic discovery, 2. Avro schema management
+and 3. Date partitioning. Data loading from Kafka to HDFS is done within a single MapReduce
+job divided into three stages:
+    1. Setup stage: Fetches topics and partitions from Zookeeper and latest offsets from Kafka.
+    2. Hadoop stage: Allocates a set of MapReduce tasks to pull data from Kafka.
+    3. Cleanup stage: Collects statistics from MapReduce tasks and submits them to Kafka for consumption by other systems.
+
+Development
+-----------
+
+To build a development version of Camus, you need to get development versions of
+`common <https://github.com/confluentinc/common>`_,
+`rest-utils <https://github.com/confluentinc/rest-utils>`_ and
+`schema-registry <https://github.com/confluentinc/schema-registry>`_ from https://github.com/confluentinc
+and install them into local Maven repository. Once the dependencies are installed, you can build
+Confluent version of Camus as follows:
+
+
+.. sourcecode:: bash
+
+    $ git clone https://github.com/confluentinc/camus.git
+    $ cd camus
+    $ mvn clean package
+
+Requirements
+------------
+- Hadoop: CDH 5.3.0
```

Comment:
Careful about wording here. As it is, it looks like you *must* use CDH 5.3.0. We require Hadoop. Maybe put the CDH part in parentheses and word it as "(We suggest CDH 5.3.0)".